### PR TITLE
MapEd: Shortcut for units stack rows inc/dec by 10/100 rows 

### PR DESCRIPTION
**Pull request by reyandme**
Also MapEd small bugfix

### CODE REVIEW

File: src/gui/pages_maped/KM_GUIMapEdUnit.pas [hunk 1]
```diff
@@ -149,8 +149,16 @@ procedure TKMMapEdUnit.Show(aGroup: TKMUnitGroup);
...
+  Unit_ArmyChangeShift(Sender, []);
+end;
+
+
+procedure TKMMapEdUnit.Unit_ArmyChangeShift(Sender: TObject; Shift: TShiftState);
+begin
+  if Sender = Button_Army_ForUp then
+    fGroup.UnitsPerRow := EnsureRange(fGroup.UnitsPerRow - GetMultiplicator(Shift), 1, 200);
+  if Sender = Button_Army_ForDown then
+    fGroup.UnitsPerRow := EnsureRange(fGroup.UnitsPerRow + GetMultiplicator(Shift), 1, 200);
```
Comment (Kromster80) [inline src/gui/pages_maped/KM_GUIMapEdUnit.pas]: `EnsureRange` looks superfluous here. `UnitsPerRow` already ensures the value is within 1..Count in its setter. Why do we need it here?

File: src/gui/pages_maped/KM_GUIMapEdUnit.pas [hunk 2]
```diff
@@ -206,6 +214,10 @@ procedure TKMMapEdUnit.Unit_ArmyChange2(Sender: TObject; Shift: TShiftState);
     NewCount := fGroup.MapEdCount + GetMultiplicator(Shift);
 
   fGroup.MapEdCount := EnsureRange(NewCount, 1, 200); //Limit max members
+
+  if (fGroup.MapEdCount < fGroup.UnitsPerRow) then
```
Comment (Kromster80) [inline src/gui/pages_maped/KM_GUIMapEdUnit.pas]: Why do we need this check here?

**Review by Kromster80 — CHANGES_REQUESTED** [review]

File: src/gui/pages_maped/KM_GUIMapEdUnit.pas (hunk 2 above)
Comment (reyandme) [reply src/gui/pages_maped/KM_GUIMapEdUnit.pas]: check out bug video on trello https://trello.com/c/u8YieijQ

File: src/gui/pages_maped/KM_GUIMapEdUnit.pas (hunk 1 above)
Comment (reyandme) [reply src/gui/pages_maped/KM_GUIMapEdUnit.pas]: fGroup.UnitsPerRow is Word type. At least we must ensure it more then 0.

**Review by Kromster80 — APPROVED** [review]

**Pull request merged**